.github/workflows/create-ci-docker/AdaptiveCpp-CI-Linux.dockerfile

```diff
@@ -0,0 +1,102 @@
+ARG BASE="ubuntu:22.04"
+
+FROM ${BASE}
+
+ARG LLVM_VERSION=18
+# make empty to skip installation of ROCM and Intel Stack
+ARG ROCM_VERSION="6.2.4"
+# make empty to skip installation of CUDA
+ARG CUDA_MAJOR_VERSION="11"
+ARG CUDA_MINOR_VERSION="0"
+ARG CUDA_PATCH_VERSION="2"
+ARG CUDA_DRIVER_VERSION="450.51.05"
```

Comment:
Why do we now need this? I don't think we've had to fix a specific driver version before? (which wouldn't matter anyway, because we don't have CUDA hardware in the github runners, so no driver needed)

Reply:
It's just a part of the installer URL ... didn't want to hardcode it down there... https://github.com/AdaptiveCpp/AdaptiveCpp/blob/f9273f1075f5ab982164388dab90350fe3ce8ce0/.github/workflows/linux.yml#L39